lib/wibox/widget/slider.lua

```diff
@@ -368,17 +409,11 @@ function slider:draw(_, cr, width, height)
     local x_offset, right_margin, y_offset = 0, 0
 
     if margins then
-        if type(margins) == "number" then
-            bar_height = bar_height or (height - 2*margins)
-            x_offset, y_offset = margins, margins
-            right_margin = margins
-        else
-            bar_height = bar_height or (
-                height - (margins.top or 0) - (margins.bottom or 0)
-            )
-            x_offset, y_offset = margins.left or 0, margins.top or 0
-            right_margin = margins.right or 0
-        end
+        bar_height = bar_height or (
+            height - (margins.top or 0) - (margins.bottom or 0)
+        )
+        x_offset, y_offset = margins.left or 0, margins.top or 0
```

Comment:
Split these assignations on two different statements. The additional `or` clauses make the line hard to read.

Reply:
Like this?
```lua
        x_offset, y_offset = margins.left or 0,
            margins.top or 0
```